examples/config.yml

```diff
@@ -15,5 +15,5 @@ gutsy:
     gem_version: 0.2.0                      # Gem version
     versions:
       - name: v1
-        resource_namespace: features          # (optional) additional URL namespacing
+        resource_url: api/features/v1       # (optional) for additional URL namespacing
```

Comment:
can we call this `path` or `namespace_path`. This doesn't contain a full URL, so it's kind of a confusing name for it. It also doesn't correspond to a single resource, but a class of resources, so I think `namespace_path` might be a less confusing name.

Reply:
I'm cool with that. I like either suggestions.

Reply:
Renamed in 0e9b600